### DISCUSSION TITLE
saving timeline Variables no longer working

### DISCUSSION THREAD

**Discussion by lauraglobig** [jspsych/jsPsych#1368] [Q&A] [answered]
Hi 

I have been using the following code successfully in the past.

 data: {Image: jsPsych.timelineVariable('image')},

However as of today it is no longer saving this variable. Is there a reason for this? I have the same issue with all other timeline Variables I am trying to save this way. 

I would appreciate any advice! 

Best

Laura

**Comment by jodeleeuw** [comment, marked as answer]
Hi Laura,

What changed in your code since it was last working? Did you change versions of jsPsych or make other changes to your code?

Are there any errors in the developer console?

**Reply by lauraglobig** [reply]
Hi Josh 
Thanks for the swift response.
The only thing I changed is that I added a second timeline variable to save, too. There are no errors and no other changes to the code. 
```js
var trial = {
...
data: {Image: jsPsych.timelineVariable('image')},
....
data: {Block: "BlockA"},
...
}
```
Would this cause an issue in terms of overwriting? 
I know I could probably combine them, too but I thought given that I can save data both in the on_finish function and also as I did here it wouldn't matter whether I keep them separate or not?

**Reply by lauraglobig** [reply]
Apologies, precisely that seems to have been the issue. I should have thought of that myself! 
Sorry for taking up your time. Thanks again!

**Reply by jodeleeuw** [reply]
Glad it's fixed! In case someone else views this in the future, the reason this doesn't work is that if you add two properties *with the same name* to the same object, then the second one overwrites the first. This isn't a jsPsych-specific thing; it's a JavaScript syntax thing.
